prisma/schema.prisma

```diff
@@ -48,13 +49,14 @@ model Comment {
 }
 
 model User {
-  id       String    @id @default(uuid())
-  email    String    @unique
-  password String    @db.VarChar(100)
-  username String    @db.VarChar(50)
-  joinedAt DateTime  @default(now())
+  id       String     @id @default(uuid())
```

Comment:
it is not yet formatted

`npx prisma format` to format it correctly

Reply:
Done

Reply:
Done